app/services/payment_provider_customers/stripe_service.rb

```diff
@@ -92,8 +92,8 @@ def check_payment_method(payment_method_id)
       result.single_validation_failure!(field: :payment_method_id, error_code: 'value_is_invalid')
     end
 
-    def generate_checkout_url
-      return result unless customer.organization.webhook_endpoints.any?
+    def generate_checkout_url(send_webhook: true)
+      return result unless (customer.organization.webhook_endpoints.any? || !send_webhook)
```

Comment:
Style/ParenthesesAroundCondition: Don't use parentheses around the condition of an `unless`. (https://rubystyle.guide#no-parens-around-condition)